src/utils/github_updater.rs

```diff
@@ -141,12 +167,21 @@ impl GithubUpdater {
             ],
         )?;
 
+        // Temporary statement to migrate production data over to the new table. Will be removed by
+        // Pietro's next PR.
+        conn.execute(
+            "UPDATE releases SET github_repo = $1 WHERE crate_id = $2;",
```

Comment:
This inherits the confusion between crates and releases from the existing codebase. Are you planning to change that in a follow-up? (This is fine for now to be clear.)

Reply:
The second PR will remove all GitHub data from crates.